test/unittests/NDB_ConfigTest.php

```diff
@@ -246,11 +246,13 @@ public function testGetProjectSettings()
         $info   = array(
                    'ProjectID'         => '999',
                    'Name'              => 'test',
+                   'Alias'             => 'TST',
```

Comment:
Class constant would be nicer

Reply:
just for that value ?

Reply:
Just if you're gonna add something new a constant would be cool. not a bit deal though, just relating to our discussion on #5432